### DISCUSSION TITLE
Setting Compass headings correctlly in Panorama Landscape?

### DISCUSSION THREAD

**Discussion by rcec52** [Stellarium/stellarium#3516] [General]
I finally got a landscape panorama installed in Stellarium 23.3. I just now need to figure out how to get the compass headings in the right direction so they are correct in the panorama landscape picture. How do I do this? 

rich :)

**Comment by gzotti** [comment]
Do you know about the User Guide? We have written it for YOU.
Stellarium User Guide 23.3, page 80, line 2.
